sdk/storage/azblob/blockblob/client_test.go

```diff
@@ -4560,6 +4560,41 @@ func (s *BlockBlobUnrecordedTestsSuite) TestLargeBlockBufferedUploadInParallel()
 	_require.Equal(*(committed[1].Size), largeBlockSize)
 }
 
+func (s *BlockBlobUnrecordedTestsSuite) TestUploadFromReader() {
+	_require := require.New(s.T())
```

Comment:
Are there any other content sizes we need to test here?

Reply:
I can think of some content sizes and some tests for that.

Reply:
Added tests for few more content sizes.